src/gdcdictionary/__init__.py

```diff
@@ -1,7 +1,17 @@
-import os
+from gdcdictionary.core import GDCDictionary, gdcdictionary, get_schema_directory
+from gdcdictionary.validators import (
+    SchemaValidator,
+    SchemaValidationError,
+    validate,
+    validate_instances,
+)
 
-from gdcdictionary.python import GDCDictionary, gdcdictionary
-
-
-
-ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+__all__ = [
```

Comment:
I haven't seen this before. Can you explain this a little?

Reply:
It's for specifying portion of code that the public should be using. Things not publicly exposed can be removed or refactored without notice, but these ones should remain the same and whenever they change we should give some sort of notice and probably deprecate first.